File: src/main/java/plus/maa/backend/common/annotation/JsonSchema.java
package plus.maa.backend.common.annotation;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * @author LoMu
 * Date  2023-01-22 17:49
 */

@Target({ElementType.METHOD})
@Retention(RetentionPolicy.RUNTIME)
public @interface JsonSchema {
    int index() default 1;
}

File: src/main/java/plus/maa/backend/common/annotation/JsonSchema.kt
package plus.maa.backend.common.annotation

/**
 * @author LoMu
 * Date  2023-01-22 17:49
 */
@Target(AnnotationTarget.FUNCTION, AnnotationTarget.PROPERTY_GETTER, AnnotationTarget.PROPERTY_SETTER)
@Retention(
    AnnotationRetention.RUNTIME
)
annotation class JsonSchema
